01-messaging-crypto-and-init.md

```diff
@@ -20,24 +20,21 @@ the unicode code point for LIGHTNING.<sup>[2](#reference-2)</sup>
 
 All messages are of form:
 
-1. `4-byte` big-endian data length.
-2. `4-byte` big-endian type.
-3. Data bytes as specified by the length.
+1. `2-byte` big-endian type.
```

Comment:
Hmm, yeah I also find this a bit ambiguous. It seems like the message header can be restricted solely to having `2/4 bytes` for the message type, with implementations relying on a lookup take which tells the how many additional bytes to read for the message.